ShoppingCartApplication_CleanCodePractices/IProduct.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ShoppingCartApplication_CleanCodePractices
+{
+    public interface IProduct
```

Comment:
Use abstract class instead of interface

Reply:
Realized that I didn't really need this Interface so removed it